common/lib/xmodule/xmodule/course_module.py

```diff
@@ -437,152 +439,308 @@ class CourseFields(object):
+                        "action_text": _("View Course Outline"),
+                        "action_external": False,
                     },
                 ],
             },
             {
-                "short_description": _("Draft Your Course About Page"),
+                "short_description": _("Learner Engagement"),
                 "items": [
                     {
-                        "short_description": _("Draft a Course Description"),
+                        "short_description": _("Send welcome email"),
+                        "long_description": _(
+                            "You agree to send a welcome email to learners on the course start date."
+                        ),
+                        "is_checked": False,
+                        "action_url": "",
+                        "action_text": "",
+                        "action_external": False,
+                    },
+                    {
+                        "short_description": _("Send emails throughout course run"),
+                        "long_description": _(
+                            "You agree to send paced emails (recommended: once a week) to students throughout the course run" 
+                            "to communicate when new content is released, to update on course events, etc."
+                        ),
+                        "is_checked": False,
+                        "action_url": "",
+                        "action_text": "",
+                        "action_external": False,
+                    },
+                    {
+                        "short_description": _("Send closing email"),
                         "long_description": _(
-                            "Courses on edX have an About page that includes a course video, description, and more. "
-                            "Draft the text students will read before deciding to enroll in your course."
+                            "You agree to send a closing email at the conclusion of the course to communicate final course details."
                         ),
                         "is_checked": False,
-                        "action_url": "SettingsDetails",
-                        "action_text": _("Edit Course Schedule &amp; Details"),
+                        "action_url": "CourseOutline",
+                        "action_text": _("View Course Outline"),
                         "action_external": False,
                     },
                     {
-                        "short_description": _("Add Staff Bios"),
+                        "short_description": _("Instruct learners to introduce themselves"),
                         "long_description": _(
-                            "Showing prospective students who their instructor will be is helpful. "
-                            "Include staff bios on the course About page."
+                            "Encourage learners to use the discussion forums to introduce themselves."
```

Comment:
On the Course Info page or in a course discussion post, ask learners to provide pertinent information about themselves .